lib/gems/pending/metadata/MIQExtract/MIQExtract.rb

```diff
@@ -158,10 +158,10 @@ def extract(category, &blk)
   end
 
   def getVMConfig(_c)
-    # Get VM config in XML format
-    config_xml = @target.vmConfig.toXML(true, @target)
-
     begin
+      # Get VM config in XML format
+      config_xml = @target.vmConfig.toXML(true, @target)
```

Comment:
I tried to add :vmConfig into SUPPORTED_METHODS array, it didn't solve the problem. All the following called methods were complaint not defined.

I agree the failures should be silent here. I'll change to raise them to caller.